packages/kovan-faucets/src/ts/id_management.ts

```diff
@@ -0,0 +1,28 @@
+import * as EthereumTx from 'ethereumjs-tx';
+
+import {configs} from './configs';
+
+type Callback = (err: Error, accounts: any) => void;
+
+export const idManagement = {
+    getAccounts(callback: Callback) {
+        /* tslint:disable */
+        console.log('configs.DISPENSER_ADDRESS', configs.DISPENSER_ADDRESS);
+        /* tslint:enable */
+        callback(null, [
+            configs.DISPENSER_ADDRESS,
+        ]);
+    },
+    approveTransaction(txData: object, callback: Callback) {
+        callback(null, true);
+    },
+    signTransaction(txData: object, callback: Callback) {
+        /* tslint:disable */
+        let tx = new EthereumTx(txData);
+        /* tslint:enable */
```

Comment:
Hm, I think this was me being lazy. We now have written type declarations for `EthereumTx` in `@0xproject/subproviders`.